lms/djangoapps/bulk_email/tests/test_course_optout.py

```diff
@@ -166,7 +166,7 @@ def create_test_message(self):
                 email_address=self.student.email,
```

Comment:
We should also probably test a couple of the edge cases here:

- empty list for the course_ids
- some course_ids are opted out of but not all
- no courses are opted out of